oak-lucene/src/test/java/org/apache/jackrabbit/oak/composite/blueGreen/CustomizedIndexTest.java

```diff
@@ -100,6 +112,8 @@ private void compositeLibs2() throws Exception {
                 "/jcr:root//*[@foo] order by @jcr:path",
                 "test-2",
                 "[/content/test, /libs/test2]");
+        assertEquals("[/oak:index/lucene, /oak:index/test-2]",
```

Comment:
There is no test case with no active indexes. Although this probably should not happen in practice, it may be good to have that test case. And also a test case with a single active index.

Reply:
Sure, I can add such a test. I don't see how this would improve code coverage or so, but it's easy, by just removing the "lucene" index.